orm/ai-sdk-nextjs/README.md

```
@@ -0,0 +1,94 @@
+npx prisma generate
+```
+
+2. Migrate the DB
+
+```
+npx prisma migrate dev --name init
+```
+
+### 4. Set up Open AI
+
+1. Navigate to https://platform.openai.com/api-keys
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)